dotcom-rendering/src/web/components/KeyEventCard.tsx

```diff
@@ -9,39 +9,50 @@ export interface KeyEvent {
 	text: string;
 	url: string;
 	format: ArticleFormat;
+	isSummary: boolean;
 }
 
-const linkStyles = css`
-	text-decoration: none;
-	line-height: 1.35;
+const linkStyles = (palette: Palette, isSummary: boolean) => {
+	const bulletColor = isSummary
+		? palette.background.keyEventBullet
+		: neutral[46];
```

Comment:
Would it be easy to bring the bullet colours for non-summary into the decide pallet too?

Reply:
Yeah maybe, at that point also the naming would need to better reflect what the colour is for here: ie `palette.background.keyEventSummaryBullet`. It's a bit long but maybe a better representation of what it actually is.

Reply:
Added all colours to `decidePalette` now 👍

Reply:
Nice one 😄